src/components/NavigationBar.tsx

```diff
@@ -0,0 +1,80 @@
+import { Nav } from "react-bootstrap";
+import {
+  Bell,
+  Bookmark,
+  CardText,
+  ChatDots,
+  House,
+  Person,
+  Hash,
+  Envelope,
+} from "react-bootstrap-icons";
+import { Link } from "react-router-dom";
+import styled from "styled-components";
+
+const StyledNav = styled(Nav)`
+  background-color: white;
+  color: black;
+  .nav-link,
```

Comment:
maybe you need to add an indent. Text too close to icon

Reply:
You also should align the list items vertically